index.html

```diff
@@ -5,22 +5,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
```

Comment:
- Kindly, the title has to adequately and briefly describes the content of the page. For example: `johnie portfolio`👌for better accessiblity👍